src/doc/examples/mapboxstyle.html

```diff
@@ -1 +1,42 @@
-<div id="map" style="width: 100%; height: 100%;" tabIndex="0"></div>
+<style>
+  button#toggle {
+    position: absolute;
+    z-index: 1;
+    top: 100px;
+    left: 10px;
+    width: 120px;
+    border: none;
+
+    /* geops theme */
+    --geops-primary-green: #76B833;
+    --ms-secondary-white: #FFFFFF;
+    --unnamed-font-family-lato: Lato;
```

Comment:
The Lato font is not loaded in the CodeSandbox as it is not integrated.

Reply:
Keep the example files as simple as possible, please. Use proper CSS and no variables and no "geOps theme" stuff.

Reply:
Done.